bundles/AdminBundle/Helper/GridHelperService.php

```diff
@@ -660,7 +660,14 @@ public function prepareAssetListingForGrid($allParams, $adminUser)
                     $language = str_replace(['none', 'default'], '', $language);
                     $conditionFilters[] = 'id IN (SELECT cid FROM assets_metadata WHERE `name` = ' . $db->quote($filterField) . ' AND `data` ' . $operator . ' ' . $value . ' AND `language` = ' . $db->quote($language). ')';
                 } else {
-                    $conditionFilters[] = $filterField . ' ' . $operator . ' ' . $db->quote($value);
+                    if ($filter['type'] == 'date' && $operator == '=') {
```

Comment:
can't we move this logic up to somewhere in between the lines 624-631 - then all date related filter logic would be together in one place.